red-hat-storage/cephci
# Adding test for scaled snap mirroring with upgrade

Automation of test case - CEPH-83574895 - Configure two-way rbd-mirror on Stand alone CEPH cluster on image (with replicated and ec pools)with snapshot based mirroring and perform upgrade

Test Steps:
    1. Bootstrap two CEPH clusters and setup snapshot based mirroring in between these clusters
    2. Create pools and images as specified, enable snapshot based mirroring for all these images
    3. Schedule snapshots for each of these images and run IOs on each of the images
    4. Perform upgrades on both the clusters
    5. Check data and run IOs again on the upgraded clusters

https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574895

Success Logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XRAIAD/
